Lib/fontTools/feaLib/builder.py

```diff
@@ -2,6 +2,7 @@
 from fontTools.misc import sstruct
 from fontTools.misc.textTools import binary2num, safeEval
 from fontTools.feaLib.error import FeatureLibError
+from fontTools.feaLib.lookupdebuginfo import LookupDebugInfo, LOOKUP_DEBUG_INFO_KEY
```

Comment:
all-lowercase for multi-word names are unusual in fonttools (or python in general); I suggest you use camelCase like most of the other modules in here